auth0_flutter/windows/test/oauth_helpers_test.cpp

```
@@ -274,13 +274,13 @@ TEST(WaitForAuthCodeCustomSchemeTest, TimeoutReturnsError) {
 TEST(WaitForAuthCodeCustomSchemeTest, CancelsWhenTokenIsAlreadyCancelled) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)